docs/conf.py

```diff
@@ -134,28 +134,28 @@ def setup(app):
     "apis-howto/api-core/metrics": "../../api-core-ug.html",
     "apis-howto/api-core/getting-started": "../../api-core-ug.html",
     "apis-howto/api-core/distributed": "../../api-core-ug.html",
-    "concepts/elastic-infrastructure": "../introduction.html",
+    "concepts/elastic-infrastructure": "../architecture/introduction.html",
     "concepts/index": "../architecture/index.html",
-    "concepts/resource-pool": "../introduction.html",
-    "concepts/scheduling": "../introduction.html",
-    "concepts/yaml": "../introduction.html",
+    "concepts/resource-pool": "../architecture/introduction.html",
+    "concepts/scheduling": "../architecture/introduction.html",
+    "concepts/yaml": "../architecture/introduction.html",
     "examples": "example-solutions/examples.html",
     "experiment-config": "reference/reference-training/experiment-config-reference.html",
-    "features/command-notebook-config": "../introduction.html",
+    "features/command-notebook-config": "../architecture/introduction.html",
     "features/commands-and-shells": "../interfaces/commands-and-shells.html",
-    "features/config-template": "../introduction.html",
-    "features/elastic-infrastructure": "../introduction.html",
+    "features/config-template": "../architecture/introduction.html",
+    "features/elastic-infrastructure": "../architecture/introduction.html",
     "features/experiments": "../training/submit-experiment.html",
-    "features/index": "../introduction.html",
+    "features/index": "../architecture/introduction.html",
     "features/job-queue": "../training/submit-experiment.html",
     "features/model-registry": "../training/model-management/model-registry-org.html",
     "features/notebooks": "../interfaces/notebooks.html",
-    "features/resource-pool": "../introduction.html",
-    "features/scheduling": "../introduction.html",
-    "features/system-architecture": "../architecture/index.html",
```

Comment:
good catch, I would have missed updating this one.

Reply:
thanks